### ISSUE TITLE
Client name for PromptForUpdatingMacTableFromDeviceCausesUuidOfMacFdBeingSearchedAndManagementMacAddressBeingReadFromMwdi

### ISSUE THREAD

**Issue by schiavi-siae** [openBackhaul/MacAddressTableRecorder#112]
In file MacAddressTableRecorder+forwardings.yaml, the client-name for the function PromptForUpdatingMacTableFromDeviceCausesUuidOfMacFdBeingSearchedAndManagementMacAddressBeingReadFromMwdi does not seem correct. It seems a OpenDayLight call.

 - forwarding-name: **PromptForUpdatingMacTableFromDeviceCausesUuidOfMacFdBeingSearchedAndManagementMacAddressBeingReadFromMwdi**
    uuid: matr-1-0-0-op-fc-is-020
    forwarding-type: InvariantProcessSnippet
    management-requests:
      operation-client-update:
      fc-port-update:
      fc-port-deletion:
      operation-client-deletion:
    initiating-requests:
      - server-name: /v1/read-current-mac-table-from-device
        uuid: matr-1-0-0-op-s-is-002
    consequent-requests:
      - client-name: **MicroWaveDeviceInventory://rests/data/network-topology:network-topology/topology=topology-netconf/node={mount-name}/yang-ext:mount/core-model-1-4:control-construct?fields=forwarding-domain(uuid;layer-protocol-name;mac-fd-1-0:mac-fd-pac(mac-fd-status(mac-address-cur)))**
        uuid: matr-1-0-0-op-c-is-mwdi-1-0-0-002

**Comment by kmohr-soprasteria** [comment]
The forwardingList needs to be corrected.
Replace 
```
client-name: MicroWaveDeviceInventory://rests/data/network-topology:network-topology/topology=topology-netconf/node={mount-name}/yang-ext:mount/core-model-1-4:control-construct?fields=forwarding-domain(uuid;layer-protocol-name;mac-fd-1-0:mac-fd-pac(mac-fd-status(mac-address-cur)))
uuid: matr-1-0-0-op-c-is-mwdi-1-0-0-002
```
with
```
client-name: MicroWaveDeviceInventory://core-model-1-4:network-control-domain=cache/control-construct={mount-name}?fields=forwarding-domain(uuid;layer-protocol-name;mac-fd-1-0:mac-fd-pac(mac-fd-status(mac-address-cur)))
uuid: matr-1-0-0-op-c-is-mwdi-1-0-0-002
```

**Comment by schiavi-siae** [comment]
Closed with issue #92
